test/TEST-50-MULTINIC/test.sh

```diff
@@ -243,7 +243,7 @@ test_setup() {
             done
         )
         inst_multiple sh shutdown poweroff stty cat ps ln ip \
-                      mount dmesg mkdir cp ping grep ls
+                      mount dmesg mkdir cp ping grep ls basename
```

Comment:
Huh? what needs basename? It's so easy to do that in bash.

Reply:
I did not know about that. Fixed now (also in #580).